src/Implementation/WarmStorage/WarmStorage/WarmStorageCoordinator.cs

```diff
@@ -56,8 +56,7 @@ private WarmStorageCoordinator(EventProcessorHost host)
             {
                 MaxBatchSize = configuration.MaxBatchSize,
                 PrefetchCount = configuration.PreFetchCount,
-                ReceiveTimeOut = configuration.ReceiveTimeout,
-                InvokeProcessorAfterReceiveTimeout = true
+                ReceiveTimeOut = configuration.ReceiveTimeout
```

Comment:
Hmm... interesting. I wonder is setting `ReceiveTimeout` is what triggers it now.
I'm still curious to know why your replaced 
``` C#
events = events ?? Enumerable.Empty<EventData>();
```
It was a guard against the null with the hope that one day the bug (I think that passing a null here is a bug) would be replaced with an empty enumerable.